packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useMappingsOperations.ts

```diff
@@ -33,29 +36,25 @@ const fetchExtractionStatus = async (
   iModelId: string,
   getAccessToken: GetAccessTokenFn,
   extractionClient: IExtractionClient,
-  setExtractionIconData: React.Dispatch<React.SetStateAction<ExtractionIconData>>,
-  setExtractionMessageData: React.Dispatch<React.SetStateAction<ExtractionMessageData[]>>
+  setExtractionStatusIcon: (extractionStatusIcon: ExtractionStatusData | ((extractionStatusIcon: ExtractionStatusData) => ExtractionStatusData)) => void,
+  setExtractionMessageData: (extractionMessageData: ExtractionMessageData[] | ((extractionMessageData: ExtractionMessageData[]) => ExtractionMessageData[])) => void
```

Comment:
Do we still need the previous state callback type?

Reply:
I have removed the callback type.

Reply:
What about the `setExtractionStatusIcon`? Do we need the previous state callback there?

Reply:
I didn't use it so I just had it removed.